mmdet3d/ops/__init__.py

```diff
@@ -3,42 +3,25 @@
                       get_compiling_cuda_version, nms, roi_align,
                       sigmoid_focal_loss)
 
-from .ball_query import ball_query
+# from .ball_query import ball_query
```

Comment:
Can remove it if it is not used?

Reply:
Sorry. This line is just for debugging. I will remove this line in the next commit.